src/components/FormNav/FormNav.js

```diff
@@ -0,0 +1,28 @@
+import NavButton from "../NavButton/NavButton";
+
+function FormNav({ handleClick, part }) {
+  if (part === 1) {
+    return (
+      <section>
+        <p>1/3</p>
+        <NavButton handleClick={handleClick} direction={"next"} />
+      </section>
+    );
+  } else if (part !== 1 && part < 4) {
+    return (
+      <section>
+        <NavButton handleClick={handleClick} direction={"prev"} />
+        <p>{part}/3</p>
+        <NavButton handleClick={handleClick} direction={"next"} />
+      </section>
+    );
+  } else if (part === 4) {
+    return (
+      <section>
+        <NavButton handleClick={handleClick} direction={"prev"} />
+      </section>
+    );
+  }
```

Comment:
Tego 'if'a można uprościć i logikę przenieść do poziomu wyżej.
Czyli to jaki jest teraz step może być określone w poziomie rodzica.
A tutaj proponuję takie rozwiązanie:
```suggestion
function FormNav({ handleClick, part, prev = false, next = true }) {
  return (
  <section>
    {prev && <NavButton handleClick={handleClick} direction={"prev"} />}
    <p>{part}/3</p>
    {next && <NavButton handleClick={handleClick} direction={"next"} />}
  </section>
  )
}
```

Reply:
done